### ISSUE TITLE
Wrapper for the rqlite status API?

### ISSUE THREAD

**Issue by rgl** [rqlite/gorqlite#42]
I would like to include the rqlite version in my application logs, but I didn't find a wrapper for the [rqlite status API](https://rqlite.io/docs/guides/monitoring-rqlite/#status-api). Can that be added, or should we do that ourselves ([as-in the rqlite cli](https://github.com/rqlite/rqlite/blob/v8.26.0/cmd/rqlite/main.go#L292-L295))?

**Comment by otoolep** [comment]
Yeah, I do not believe it's exposed via the client library. It's easy to add it, PRs very welcome.

**Comment by otoolep** [comment]
You can also access the version of rqlite via any HTTP request, as it's added as the HTTP header `X-RQLITE-VERSION` in all response by `rqlited`.

**Comment by zhangyongding** [comment]
I think it's possible to consider returning the versions of all nodes in the nodes interface. because X-RQLITE-VERSION only returns the version of the current node
